.circleci/config.yml

```diff
@@ -95,7 +95,62 @@ jobs:
             - run: make publish-<<parameters.image-type>> NGC_PUBLISH= REGISTRY_REPO=environments-dev
       - store_artifacts:
           path: /tmp/artifacts
+  publish-docker-multiplatform:
+    parameters:
+      image-type:
+        type: string
+      dev-mode:
+        type: boolean
+        default: false
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 19.03.12
```

Comment:
why is it `setup_remote_docker` here, and `install-docker` in the old build?

Reply:
This job is only publishing to docker repositories.
The other jobs use a `machine` executor and manually install docker on it using `install-docker` - not sure why.
For the ARM image, I think it actually makes sense to use a `machine` executor as it allows me to specify an ARM machine and build natively, without emulation. As I mentioned in my response to another one of your comments, I have been unable to build multi-platform images at once with `buildx` due to `qemu` issues.